lnrpc/rpc.swagger.json

```diff
@@ -17,7 +17,8 @@
   "paths": {
     "/v1/balance/blockchain": {
       "get": {
-        "summary": "* lncli: `walletbalance`\nWalletBalance returns total unspent outputs(confirmed and unconfirmed), all\nconfirmed unspent outputs and all unconfirmed unspent outputs under control\nof the wallet.",
+        "summary": "*\nGenSeed is the first method that should be used to instantiate a new lnd\ninstance. This method allows a caller to generate a new aezeed cipher seed\ngiven an optional passphrase. If provided, the passphrase will be necessary\nto decrypt the cipherseed to expose the internal wallet seed.",
```

Comment:
This doesn't look right. The description for `WalletBalance` is being overwritten with `GenSeed`'s.

Reply:
@wpaulino 

Thank you for catching this. Investigating. It seems that the root cause is https://github.com/grpc-ecosystem/grpc-gateway/issues/688

Reply:
Yeah, need to figure out why this is happening...

Reply:
@halseth 

The easiest solution is to lock `grpc-gateway` to the latest stable `v1.4.1`.